Swiftly/Swiftly.swift

```diff
@@ -34,7 +34,7 @@ public extension Swiftlyable {
 
      - returns: An array of constraints that represent the applied layout. This can be used to dynamically enable / disable a given layout.
      */
-    internal func applyLayout(layoutArray: [Swiftly]) -> [NSLayoutConstraint] {
+    internal func applyLayout(layoutArray: [Swiftly], applyImmediately: Bool) -> [NSLayoutConstraint] {
```

Comment:
Instead of doing this `applyImmediately` parameter here, I'd rather change this to `createLayout` and have the `applyLayout` functions pass the result to `activate(constraints:)`